sphinx/ext/autodoc/type_comment.py

```diff
@@ -41,7 +41,18 @@ def get_type_comment(obj: Any) -> ast.FunctionDef:
             subject = cast(ast.FunctionDef, module.body[0])  # type: ignore
 
         if getattr(subject, "type_comment", None):
-            return ast_parse(subject.type_comment, mode='func_type')  # type: ignore
+            comment = subject.type_comment  # type: ignore
+            if comment.startswith("(...)"):
+                # resolve parameters individually.
+                args_comment = [
+                    (arg.type_comment or "Any")  # type: ignore
+                    for arg in subject.args.args
```

Comment:
There some kind of arguments (ex. posonlyargs, vararg, etc.). So they would be supported.